unit-tests/py/rspy/test.py

```diff
@@ -855,6 +859,31 @@ def run( self, command, timeout=5, on_fail=RAISE ):
             self.wait_until_ready( timeout=timeout, on_fail=on_fail )
 
 
+    def capture_stdout( self ):
+        """
+        Start collecting stdout from the remote, if not already collecting it.
+        Stdout collected will not be printed to our stdout, and will instead be returned in get_stdout()
+        as a list of lines. Debug lines (-D-...) are filtered out.
```

Comment:
Can we give an option to collect also debug messages? What if it's combined with `--debug` flag?

Reply:
I don't see a reason at this time, as debug messages are not a way of outputting anything, and are also unreliable as they are many times "randomly" interspersed with stdout because they happen from threads, etc.